src/Queue/QueueFactory.php

```diff
@@ -15,11 +16,19 @@ class QueueFactory {
 
   const SERVICE_NAME = 'queue.rabbitmq';
```

Comment:
Not sure if service name should be put in a class constant as it is defined outside of this class (in `rabbitmq.services.yml`).